src/main/java/core/utils/Log.java

```diff
@@ -1,8 +1,19 @@
+/*
+ * @(#) FormPageFour.java Copyright (c) 2019 Jala Foundation.
```

Comment:
change the name of FormPageFour.java accord to your classes

Reply:
Solved.